app/command_utils.cc

```diff
@@ -0,0 +1,237 @@
+        ADD_UNTRANSLATED_COMMAND(ZOOM_MINUS),
+        ADD_UNTRANSLATED_COMMAND(ZOOM_PERCENT_DISPLAY),
+
+        // Focus
+        ADD_UNTRANSLATED_COMMAND(FOCUS_TOOLBAR),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_LOCATION),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_SEARCH),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_MENU_BAR),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_NEXT_PANE),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_PREVIOUS_PANE),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_BOOKMARKS),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_INACTIVE_POPUP_FOR_ACCESSIBILITY),
+        ADD_UNTRANSLATED_COMMAND(FOCUS_WEB_CONTENTS_PANE),
+
+        // UI bits
+        ADD_UNTRANSLATED_COMMAND(OPEN_FILE),
+        ADD_UNTRANSLATED_COMMAND(CREATE_SHORTCUT),
+        ADD_UNTRANSLATED_COMMAND(DEVELOPER_MENU),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_CONSOLE),
+        ADD_UNTRANSLATED_COMMAND(TASK_MANAGER),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_DEVICES),
+        ADD_UNTRANSLATED_COMMAND(FEEDBACK),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BOOKMARK_BAR),
+        ADD_UNTRANSLATED_COMMAND(SHOW_HISTORY),
+        ADD_UNTRANSLATED_COMMAND(SHOW_BOOKMARK_MANAGER),
+        ADD_UNTRANSLATED_COMMAND(SHOW_DOWNLOADS),
+        ADD_UNTRANSLATED_COMMAND(CLEAR_BROWSING_DATA),
+        ADD_UNTRANSLATED_COMMAND(IMPORT_SETTINGS),
+        ADD_UNTRANSLATED_COMMAND(OPTIONS),
+        ADD_UNTRANSLATED_COMMAND(EDIT_SEARCH_ENGINES),
+        ADD_UNTRANSLATED_COMMAND(VIEW_PASSWORDS),
+        ADD_UNTRANSLATED_COMMAND(ABOUT),
+        ADD_UNTRANSLATED_COMMAND(HELP_PAGE_VIA_KEYBOARD),
+        ADD_UNTRANSLATED_COMMAND(SHOW_APP_MENU),
+        ADD_UNTRANSLATED_COMMAND(MANAGE_EXTENSIONS),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_INSPECT),
+        ADD_UNTRANSLATED_COMMAND(BOOKMARKS_MENU),
+        ADD_UNTRANSLATED_COMMAND(SHOW_AVATAR_MENU),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_REQUEST_TABLET_SITE),
+        ADD_UNTRANSLATED_COMMAND(DEV_TOOLS_TOGGLE),
+        ADD_UNTRANSLATED_COMMAND(TAKE_SCREENSHOT),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_FULLSCREEN_TOOLBAR),
+        ADD_UNTRANSLATED_COMMAND(INSTALL_PWA),
+        ADD_UNTRANSLATED_COMMAND(PASTE_AND_GO),
+        ADD_UNTRANSLATED_COMMAND(SHOW_FULL_URLS),
+        ADD_UNTRANSLATED_COMMAND(CARET_BROWSING_TOGGLE),
+        ADD_UNTRANSLATED_COMMAND(TOGGLE_QUICK_COMMANDS),
+
+        // Media
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_PLAYPAUSE),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_MUTE),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_LOOP),
+        ADD_UNTRANSLATED_COMMAND(CONTENT_CONTEXT_CONTROLS),
+
+#if BUILDFLAG(ENABLE_SCREEN_AI_SERVICE)
+        // Screen AI Visual Annotations.
+        ADD_UNTRANSLATED_COMMAND(RUN_SCREEN_AI_VISUAL_ANNOTATIONS),
+#endif
+
+        // Tab search
+        ADD_UNTRANSLATED_COMMAND(TAB_SEARCH),
+        ADD_UNTRANSLATED_COMMAND(TAB_SEARCH_CLOSE),
+
+        // Brave Commands
+        ADD_UNTRANSLATED_COMMAND(SHOW_BRAVE_REWARDS),
```

Comment:
I think we need to double check whether all commands are guarded properly.
Ex, using `BUILDFLAG(ENABLE_TOR)` for tor specific commands such as `NEW_TOR_CONNECTION_FOR_SITE` or `NEW_OFFTHERECORD_WINDOW_TOR`.
All VPN commands also should be guarded `BUILDFLAG(ENABLE_BRAVE_VPN`.

Reply:
Hmm, I think `AllCommandsShouldBeExecutableWithoutCrash` seems difficult to run all commands because maybe some commands need to enable featur flag also?

Reply:
> I think we need to double check whether all commands are guarded properly. Ex, using `BUILDFLAG(ENABLE_TOR)` for tor specific commands such as `NEW_TOR_CONNECTION_FOR_SITE` or `NEW_OFFTHERECORD_WINDOW_TOR`. All VPN commands also should be guarded `BUILDFLAG(ENABLE_BRAVE_VPN)`.

Oh, NVM. webui fiters enabled commands by `chrome::SupportsCommand()`.